Jenkinsfile

```diff
@@ -148,11 +149,15 @@ stage('Prepare') {
   }
 }
 
-stage("Sanity Check") {
+stage('Sanity Check') {
   timeout(time: max_time, unit: 'MINUTES') {
     node('CPU') {
-      ws(per_exec_ws("tvm/sanity")) {
+      ws(per_exec_ws('tvm/sanity')) {
         init_git()
+        docs = sh (returnStatus: true, script: '''
```

Comment:
could we name this `is_docs_only_build`?

Reply:
I have a comment regarding to this naming as well, but I prefer yours :D